frontend/src/Components/Home/PopularArticles/PopularArticleItems/PopularArticleItem.jsx

```diff
@@ -0,0 +1,14 @@
+import { Stack, Image } from "react-bootstrap";
+import "./PopularArticleItem.scss";
+import ArticleImage from "../ArticleImage/ArticleImage";
+
+export default function PopularArticleItem({ title, article_img, number }) {
+  return (
+    <>
+      <Stack gap={3} className="d-flex align-items-center p-3">
+        <ArticleImage article_img={article_img} number={number} />
+        <a className="text-center fw-medium popular-article-title">{title}</a>
+      </Stack>
```

Comment:
If you look in the figma you can see that the words only expand a little wider than the image, so you could set a max width for them. this isn't really a big deal though